unity-renderer/Assets/DCLPlugins/ECS7/Systems/SceneBoundsCheckerSystem/OutOfSceneBoundsFeedbackStyle/ECSOutOfSceneBoundsFeedback_EnabledToggle.cs

```diff
@@ -0,0 +1,41 @@
+
+using DCL.ECS7.InternalComponents;
+using DCL.ECSRuntime;
+
+namespace ECSSystems.ECSSceneBoundsCheckerSystem
+{
+    public class ECSOutOfSceneBoundsFeedback_EnabledToggle : IECSOutOfSceneBoundsFeedbackStyle
+    {
+        public void ApplyFeedback(ECSComponentData<InternalSceneBoundsCheck> sbcComponentData, IECSReadOnlyComponentData<InternalVisibility> visibilityComponentData, bool isInsideBounds)
+        {
+            if (sbcComponentData.model.renderers != null)
+            {
+                int count = sbcComponentData.model.renderers.Count;
+                for (var i = 0; i < count; i++)
+                {
+                    sbcComponentData.model.renderers[i].enabled = isInsideBounds
+                                                                  && (visibilityComponentData == null
```

Comment:
method definition is not that big. we would prefer to keep it as it is since if we extracted to a local method anyone reading the code would have to jump between definitions to understand what is the code about, if we keep it as it is, it's all readable without even need to scroll down